lib/active_interactor/organizer/interactor_interface.rb

```diff
@@ -78,8 +83,31 @@ def execute_inplace_callback(target, callback)
         resolve_option(target, callbacks[callback])
       end
 
+      def execute_after_perform_callbacks(context)
+        return unless after_perform_callbacks.present?
+
+        interactor = interactor_class.new(context)
+        env = ActiveSupport::Callbacks::Filters::Environment.new(interactor, false, nil)
+        after_perform_callbacks.compile.invoke_after(env)
+        interactor.send(:context)
+      end
+
       private
 
+      def init_after_perform_callbacks
+        after_callbacks_deferred = @interactor_class.present? 
```

Comment:
Can we use the reader here:

https://github.com/trainual/activeinteractor/blob/86a12c4468364d2d3e09227431d90026b27181e6/lib/active_interactor/organizer/interactor_interface.rb#L37

instead of `@`? and the lines below?